accounting_reporting_core/src/main/java/org/cardanofoundation/lob/app/accounting_reporting_core/repository/CustomTransactionRepository.java

```diff
@@ -2,12 +2,24 @@
 
 import org.cardanofoundation.lob.app.accounting_reporting_core.domain.core.TransactionType;
 import org.cardanofoundation.lob.app.accounting_reporting_core.domain.core.ValidationStatus;
+import org.cardanofoundation.lob.app.accounting_reporting_core.domain.entity.ReconcilationRejectionCode;
 import org.cardanofoundation.lob.app.accounting_reporting_core.domain.entity.TransactionEntity;
+import org.cardanofoundation.lob.app.accounting_reporting_core.resource.requests.ReconciliationFilterRequest;
+import org.cardanofoundation.lob.app.accounting_reporting_core.resource.requests.ReconciliationFilterStatusRequest;
 
 import java.util.List;
+import java.util.Set;
 
 public interface CustomTransactionRepository {
     List<TransactionEntity> findAllByStatus(String organisationId,
                                             List<ValidationStatus> validationStatuses,
                                             List<TransactionType> transactionType);
+
+    public List<Object[]> findAllReconciliationSpecial(ReconciliationFilterRequest body);
```

Comment:
Those requests are coming from the view layer, is there any chance that here we could simply pass parameters extracted from the request than create a dependency of repository later to the view layer (which is far from ideal and anti-pattern)?